### ISSUE TITLE
@import-ed SCSS in other SCSS files do not hotload

### ISSUE THREAD

**Issue by MrBrax** [Facepunch/sbox-issues#5317]
### Describe the bug

Simple, imported SCSS doesn't hotload when saving. The "master" SCSS file has to be saved for the child one to update.

### To Reproduce

1. Make two scss files
2. @import the second one in the first
3. Try hotloading the imported one

### Expected behavior

To properly hotload imported ones

**Comment by MrBrax** [comment]
https://github.com/user-attachments/assets/5efd8862-4297-4467-bdb0-8d8e91d4f77e

Issue shown in video

**Comment by CarsonKompon** [comment]
I had fixed this but it still wasn't working for relative paths. Should be all good now :)

**Comment by MrBrax** [comment]
nope still not working for me, i'm importing inside a stylesheet like `@import "./shared.scss";` from a panel component with the stylesheet attribute

**Comment by CarsonKompon** [comment]
Yeah sorry about that I had to revert the changes because it broke something else. I'll need to take another look today :P

**Comment by MrBrax** [comment]
my sanity levels are already increasing, thanks much

**Comment by Tankonyako** [comment]
hot reloading of `@import`ed scss are not working in PanelComponent, but in Panel all works fine

**Comment by MrBrax** [comment]
i'm also noticing some weird stuff going on with hotloading again

**Comment by CarsonKompon** [comment]
Yeah I believe this commit is what brought it back:
![Image](https://github.com/user-attachments/assets/ee2383ec-71a6-4c07-804f-2b9c22f252a2)

**Comment by MrBrax** [comment]
any news on this?

**Comment by CarsonKompon** [comment]
I’ll try taking another look today

**Comment by CarsonKompon** [comment]
I just opened my test project with a repro of this and it seems to be working already. Can you confirm for me (or send a repro of your own if you're still encountering it)?

**Comment by handsomematt** [comment]
It'd be great if we got some unit tests for this @CarsonKompon I feel like it breaks every other month

**Comment by MrBrax** [comment]
@CarsonKompon i'll try to make a repro, but i have it set up like this:

BankPanel.razor
BankPanel.razor.cs
BankPanel.razor.scss -> import("../Shared.scss") -> Shared.scss

**Comment by CarsonKompon** [comment]
@MrBrax Just pushed a new commit that should solve the hotloading issues, let me know if this works for you 🤞

**Comment by MrBrax** [comment]
seems to work, thanks carson!

**Comment by MrBrax** [comment]
i spoke too soon, imported files in that imported files don't hotload

**Comment by MrBrax** [comment]
alright it's working sometimes, i really need to figure out what's going on

**Comment by MrBrax** [comment]
yep it still happens in imported ones
